modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+= CloudHub 2.0 REST API
+
+*Last update* 02-Mar-22
+
+Added all the endpoints from https://github.com/mulesoft/rtf-management-service/blob/master/assets/raml/public/api.raml[GitHub].
+
+* *Private Spaces*
+** Get the list of private spaces in the organization.
+** Create a private space.
+** Delete the private space.
+** Get the private space. *What does this mean?*
+** Update an existing private space.
+
+* *Connections*
+** Get the list of connections inside the private space.
+** Create a private space connection. *What is this?*
+** Delete the private space connection. 
+** Get the private space connection.
+** Update an existing private space connection.
+** Download the connection guide for a VPN connection.
+
+* *VPN*
+** Add a VPN to an existing connection.
+** Delete a VPN from a connection.
+** Update a VPN connection.
+** Get supported VPN configurations.
+
+
+* *Transit Gateway Attachments*
+** Get list of TGW connections associated with a private space.
+** Create a private space TGW connection.
+** Delete the private space TGW connection.
+** Update the private space TGW connection.
+** Update routes for a TGW connection.
+** Get the routes configured for the TGW connection.
+** Get the TGW routes configured for a private space.
+** Get list of TGW connections assocated with an organization.
+** Remove private space TGW connection.
+** Get TGW routes configured for a gateway.
+
+* *Log Forwarding*
+** Get the log forwarding configuration information.
+** Update the log forwarding configuration.
+** Send test message to validate the end-to-end connectivity.
+** Get the supported external log forwarding schemas.
+** Get the specific configuration of a supported external log forwarding schema.
+** Get the log forwarding config information.
+** Update the Anypoint log forwarding config.
+** Deletes external and Anypoint Monitoring log forwarding configuration.
+** Configure external log forwarding.
+** Deletes the external log forwarding configuration.
+
+* *TLS Contexts*
+** Get the list of TLS contexts inside the private space.
+** Create a TLS context.
+** Delete the TLS context.
+** Get the TLS context.
+** Update an existing TLS context.
+
+* *Runtime Fabrics*
+** Get the list of Runtime Fabrics in the organization.
+** Runtime Fabric register endpoint.
+** Delete associated apps.
+** Get capacity information about the runtime fabric for an environment.
+** Update existing fabric.
+** Get a detailed health check for the Runtime Fabric.
+** Get the list of Runtime Fabric Groups.
+** Get the list of environment associations.
+** Create a new environment association.
+** Update an existing environment association.
+** Retrieve short-lived credential to allow pulling Runtime Fabric Docker images.
+** Retrieve properties related to image registry.
```

Comment:
We don't need to mention Runtime Fabrics in CH2.0 docs. It's a separate product.